src/components/Books.js

```diff
@@ -1,16 +1,35 @@
 /* eslint-disable max-len */
+/* eslint-disable no-restricted-syntax */
+/* eslint-disable guard-for-in */
```

Comment:
- [ ] Please remove all the **eslint disable** statement. Because ` Eslint` helps developers to write clean JavaScript codes and adapt best practices when writing the code.  You just need to export it as the default code. :smile: